packages/notebook-extension/src/index.ts

```diff
@@ -70,6 +70,9 @@ namespace CommandIDs {
   const switchKernel = 'notebook:switch-kernel';
 
   export
+  const createConsoleForNotebook = 'notebook:create-console-for-notebook';
```

Comment:
I would call the command `notebook:create-console` to match the naming of the similar command in the fileeditor.